apps/website-25/src/components/lander/ClassicHero.tsx

```diff
@@ -80,7 +80,7 @@ interface ClassicHeroProps {
 
 const ClassicHero = ({ children, className = '' }: ClassicHeroProps) => {
   return (
-    <HeroSection className={`-mt-20 text-white bg-[url('/images/logo/logo_hero_background.svg')] bg-cover ${className}`}>
+    <HeroSection className={`text-white bg-[url('/images/logo/logo_hero_background.svg')] bg-cover ${className}`}>
```

Comment:
This feels dangerous, but actually it's only used on the classic lander which itself is suffering from a bug because of this margin!

https://website-25-staging.k8s.bluedot.org/lander/classic

This change fixes this and makes it easier to use with our main Nav too :)